### DISCUSSION TITLE
Mixed tax products with extra text on the invoice

### DISCUSSION THREAD

**Discussion by ginkel** [pretix/pretix#3912] [Q&A] [answered]
Let's say, we have a product that consists of a taxable portion and a tax-free portion (e.g., the accommodation at a German youth hostel). Which configuration would you recommend to have the appropriate VAT rates on the invoice?

I thought about using a bundled product that has a VAT rate of 0% assigned to it, but we need an additional text on the invoice ("Im Namen und auf Rechnung von XYZ, Steuernummer 123/456/7890").

Any ideas are very much appreciated!

Thanks!

**Comment by pc-coholic** [comment]
For now, that's probably the only way to go.

If you need additional text on the invoice, you can add to the invoice customization options. (Settings -> Invoice)

**Reply by ginkel** [reply]
Thanks for the speedy reply! :-) If I'm not mistaken, the invoice customization is global per event. Is there a way to display an extra text on the invoice with the respective position or when a tax rule is used?

**Reply by raphaelm** [reply, marked as answer]
There actually is, with using a custom tax rule that has an extended rule set, you can set a special invoice text.

**Reply by ginkel** [reply]
Excellent, thanks a lot!
